packages/theme-dark-extension/style/variables.css

```
@@ -167,6 +167,8 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-content-font-color2: rgba(255, 255, 255, 0.7);
   --jp-content-font-color3: rgba(255, 255, 255, 0.5);
   --jp-content-link-color: var(--md-blue-300);
+  --jp-content-link-visited-color: rgba(255,255,255, 0.3);
+  --jp-content-link-hover-color: var(--md-blue-400)
```

Educated guess that this is why the lint check is failing.

```suggestion
  --jp-content-link-hover-color: var(--md-blue-400);
```